content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -197,6 +197,33 @@ As Pod specs with GMSA fields populated (as described above) are applied in a cl
 
 1. The container runtime configures each Windows container with the specified GMSA credential spec so that the container can assume the identity of the GMSA in Active Directory and access services in the domain using that identity.
 
+## Containerd 
+
+On Windows Server 2019, in order to use GMSA with Containerd, you must be running OS Build 17763.1817 (or later) which can be installed using the patch [KB5000822](https://support.microsoft.com/en-us/topic/march-9-2021-kb5000822-os-build-17763-1817-2eb6197f-e3b1-4f42-ab51-84345e063564). 
+
+There is also a known issue with containerd when trying to connect to SMB shares from Pods. Once you have configured GMSA, the pod will be unable to connect to the share using the Hostname or FQDN but connecting via IP address works as expected. 
+
+```shell
+Windows PowerShell
+Copyright (C) Microsoft Corporation. All rights reserved.
+PS C:\> ping adserver.ad.local
+Pinging adserver.ad.local [192.168.111.18] with 32 bytes of data:
+Reply from 192.168.111.18: bytes=32 time=6ms TTL=124
+Reply from 192.168.111.18: bytes=32 time=5ms TTL=124
+...
+PS C:\> cd \\adserver.ad.local\test
+cd : Cannot find path '\\adserver.ad.local\test' because it does not exist.
+At line:1 char:1
++ cd \\adserver.ad.local\test
++ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    + CategoryInfo          : ObjectNotFound: (\\adserver.ad.local\test:String) [Set-Location], ItemNotFoundException
+    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand
+PS C:\> cd \\192.168.111.18\test
+PS Microsoft.PowerShell.Core\FileSystem::\\192.168.111.18\test>
+```
```

Comment:
This is not valid Bourne shell.

Please:
- mark the code block as PowerShell
- [omit the prompt](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt)
- [separate commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)

Reply:
thanks for the feedback @sftim I have corrected those and I also went ahead and fixed the other parts that were not in the correct style as well, please can you take another look?